src/components/grid/Grid.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { ReactNode } from 'react';
+import './grid.css';
+
+interface IGridProps {
+  className?: string;
+  template_rows?: string;
+  template_columns?: string;
+  gap?: string;
+  children?: ReactNode;
```

Comment:
this is not optional 
gap is not optional
template_columns is not optional 
className is optional